mcs/class/System.XML/Dummy.cs

```diff
@@ -0,0 +1,121 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Xml.Schema;
+
+namespace System.Xml
```

Comment:
No need to re-implement this. Include mcs/build/common/SR.cs instead as we do elsewhere